src/org/labkey/test/tests/TimeChartAPITest.java

```diff
@@ -311,13 +312,13 @@ private void testVisApi(File htmlPage, String[] testTitles, @Nullable int[] test
                 waitForElement(Locator.paginationText(testRowCounts[testIndex]), WAIT_FOR_JAVASCRIPT);
             }
 
-            DataRegionTable table = null;
+            CachingSupplier<DataRegionTable> table = new CachingSupplier<>(
+                () -> new DataRegionTable("apiTestDataRegion", this));
```

Comment:
Maybe add a comment here why using a CachingSupplier?

Reply:
Actually, just using `DataRegionFinder.findWhenNeeded` would be more appropriate.